diff --git a/MANIFEST.in b/MANIFEST.in
new file mode 100644
index 0000000..02452a3
--- /dev/null
+++ b/MANIFEST.in
@@ -0,0 +1,26 @@
+include ceilometer/storage/sqlalchemy/migrate_repo/versions/021_sqlite_upgrade.sql
+include ceilometer/storage/sqlalchemy/migrate_repo/versions/023_sqlite_upgrade.sql
+include ceilometer/storage/sqlalchemy/migrate_repo/migrate.cfg
+include ceilometer/locale/ceilometer-log-error.pot
+include ceilometer/locale/zh_CN/LC_MESSAGES/ceilometer-log-info.po
+include ceilometer/locale/zh_CN/LC_MESSAGES/ceilometer.po
+include ceilometer/locale/zh_CN/LC_MESSAGES/ceilometer-log-error.po
+include ceilometer/locale/zh_CN/LC_MESSAGES/ceilometer-log-warning.po
+include ceilometer/locale/it/LC_MESSAGES/ceilometer.po
+include ceilometer/locale/pt_BR/LC_MESSAGES/ceilometer.po
+include ceilometer/locale/ko_KR/LC_MESSAGES/ceilometer.po
+include ceilometer/locale/ru/LC_MESSAGES/ceilometer.po
+include ceilometer/locale/ceilometer-log-info.pot
+include ceilometer/locale/es/LC_MESSAGES/ceilometer.po
+include ceilometer/locale/en_GB/LC_MESSAGES/ceilometer.po
+include ceilometer/locale/en_GB/LC_MESSAGES/ceilometer-log-error.po
+include ceilometer/locale/zh_TW/LC_MESSAGES/ceilometer.po
+include ceilometer/locale/fr/LC_MESSAGES/ceilometer-log-info.po
+include ceilometer/locale/fr/LC_MESSAGES/ceilometer.po
+include ceilometer/locale/ja/LC_MESSAGES/ceilometer.po
+include ceilometer/locale/ceilometer-log-warning.pot
+include ceilometer/locale/ceilometer.pot
+include ceilometer/locale/de/LC_MESSAGES/ceilometer.po
+include ceilometer/meter/data/meters.yaml
+include ceilometer/api/app.wsgi
+include ceilometer/hardware/pollsters/data/snmp.yaml
