diff --git a/ceilometer/service.py b/ceilometer/service.py
index aac7242..b1478d7 100644
--- a/ceilometer/service.py
+++ b/ceilometer/service.py
@@ -121,7 +121,6 @@ def prepare_service(argv=None, config_files=None):
     if argv is None:
         argv = sys.argv
     cfg.CONF(argv[1:], project='ceilometer', validate_default_values=True,
-             version=version.version_info.version_string(),
              default_config_files=config_files)
     log.setup(cfg.CONF, 'ceilometer')
     # NOTE(liusheng): guru cannot run with service under apache daemon, so when
